js/Timer_JS_5-6.js

```diff
@@ -0,0 +1,136 @@
+var oTimer = {
+
+  iStartTime: 0,
+  iTimeNow: 0,
+  iPrintTime: 0,
+  iStopTime: 0,
+  iSplitTime: 0,
+  timerId: 0,
+  bActiveTimer: false,
+
+
+  /**
+   * sets timer's time of start
+   */
+  startTimer: function () {
+    iStartTime = Date.now() - this.iPrintTime;
```

Comment:
you are using global variable instead of object property here. use this.iStartTime

Reply:
done